Cargo.toml

```diff
@@ -170,7 +170,7 @@ warp-embed = { version = "0.4", default-features = false }
 wascap = { version = "0.12", path = "./crates/wascap", default-features = false }
 wash-cli = { version = "0", path = "./crates/wash-cli", default-features = false }
 wash-lib = { version = "0.15", path = "./crates/wash-lib", default-features = false }
-wasi-common = { version = "15", default-features = false }
+wasi-common = { branch = "release-16.0.0", git = "https://github.com/bytecodealliance/wasmtime", default-features = false }
```

Comment:
These branch pins are going to make it so we can't cut crates releases. How long are we planning on staying pinned to the branch?